b_cfn_elasticsearch_cloner/resource.py

```diff
@@ -30,11 +35,25 @@ def __init__(
         elasticsearch_index: ElasticsearchIndexResource,
         dynamodb_table: Table,
         kms_key: Optional[Key] = None,
+        sagemaker_endpoint_name: str = None,
+        sagemaker_embeddings_key: str = None,
+        sagemaker_arn: str = None
     ) -> None:
         super().__init__(scope=scope, id=id)
 
         elasticsearch_layer = BElasticsearchLayer(scope=self, name=f"{id}ElasticsearchLayer")
 
+        if bool(sagemaker_endpoint_name) ^ bool(sagemaker_embeddings_key):
+            raise ValueError(
+                f'To use sentence embedding, all of the following variables are required: '
```

Comment:
clarify message that none of the parameters could be used for no sagemeker stuff